app/views/nut_job_authentication/edit_profile_with_errors.html.erb

```diff
@@ -0,0 +1,52 @@
+<div >
+  <h5>
+    Edit Nut Job Account
+  </h5>
+
+  <% if @nut_job.errors.any? %>
+  <% @nut_job.errors.full_messages.each do |message| %>
+    <div class="alert">
```

Comment:
Rendering an error form seems new— or at least different than the previous version. I think we would need to kind of explain what this is doing, which is perhaps what we'll do today.